src/components/Footer/SoilFooterWithContacts.tsx

```diff
@@ -5,10 +5,16 @@ import plushLogo from '@/assets/images/footer-logo.png'
 import { Link } from 'react-router-dom'
 
 export const SoilFooterWithContacts: FC = () => {
+
+  const PLUSH_LINK = window.config.PLUSH_LINK ?? import.meta.env.VITE_PLUSH_LINK
```

Comment:
Not necessary, but would be nice to name it PLUSH_WEBSITE_LINK to make a bit clearer.